src/domain/참고하는/components/index.tsx

```diff
@@ -38,15 +41,17 @@ const 참고하는TabContent = () => {
         selectedFilterButton={selectedFilterButton}
         handleFilterButtonSelect={handleFilterButtonSelect}
       />
-      <ul className={styles.referCardsWrapper}>
-        {data?.content.map((data) => (
-          <참고하는TemplateCard
-            key={data.templateId}
-            data={data}
-            memoFolders={memoFoldersData}
-          />
-        ))}
-      </ul>
+      <ResponsiveMasonry columnsCountBreakPoints={{ 768: 2, 1080: 3 }}>
+        <Masonry className={styles.referCardsWrapper} gutter="16px">
+          {data.content.map((data) => (
```

Comment:
어떤 코드인지 불명확해서...! 아래처럼 변경되면 좋을 것 같아요
```suggestion
  const { data: templates } = useGetTemplates({
    category: selectedFilterHeader,
    sort: selectedFilterButton,
  });
  
{templates.content.map((template) => ()}
```

Reply:
해당 부분 반영해보았습니다 감사합니다 ㅎㅎ